docs/ja/faq.md

```
@@ -5,87 +5,86 @@ outline: deep
-- [セキュリティ](#セキュリティ)
 
----
-
-## データ量とパフォーマンス
+* [データ量とパフォーマンス](#データ量とパフォーマンス)
+  
+* [FIWARE Orion との違い](#fiware-orion-との違い)
+  
+* [デプロイと運用](#デプロイと運用)
+  
+* [API の使用方法](#how-to-use-the-api)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)